spec/support/govuk_i18n_cov.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class GOVUKI18nCov
+  module I18n
+    module Registry
```

Comment:
Any reason this should be two modules deep?

Reply:
Nope - you find the flaws in my pasted code ;-). I've revised it now, which I think has made it a bit clearer too!